### DISCUSSION TITLE
Παραδοτέο 8 1Β

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1657] [Announcements]
Για το 1Β η αναφορά του case study σε πιο αποθετήριο θα γίνει upload στο _cast-study ή στο extras?

**Comment by p15bako** [comment]
Νομίζω πως στο _case-study θα βάλεις το πινακάκι και στο extras θα γράψεις την αναλυτική μελέτη περίπτωσης που συνδιάζει τις θεματικές ενότητες που ασχολήθηκες στα παραδοτέα συμμετοχικού περιεχομένου Α1 και Α2.
